automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -61,6 +61,9 @@ def check_necessary_env_variables_for_upgrade(product):
     if not os.environ.get('OS'):
         failure.append('Please provide OS version as rhel7 or rhel6, '
                        'And retry !')
+    if not os.environ.get('SATELLITE_HOSTNAME'):
```

Comment:
This is optional and not required. 
If customer wants to run upgrade on his own sat then only he has to provide SATELLITE_HOSTNAME.
Else this is not required.